src/Npgsql.NodaTime/IntervalHandler.cs

```diff
@@ -19,10 +19,13 @@ public override NpgsqlTypeHandler<Period> Create(PostgresType postgresType, Npgs
                 : throw new NotSupportedException($"The deprecated floating-point date/time format is not supported by {nameof(Npgsql)}.");
     }
 
-    sealed class IntervalHandler : NpgsqlSimpleTypeHandler<Period>, INpgsqlSimpleTypeHandler<NpgsqlTimeSpan>, INpgsqlSimpleTypeHandler<TimeSpan>
+    sealed class IntervalHandler : NpgsqlSimpleTypeHandler<Period>,
+        INpgsqlSimpleTypeHandler<Duration>, INpgsqlSimpleTypeHandler<NpgsqlTimeSpan>, INpgsqlSimpleTypeHandler<TimeSpan>
```

Comment:
Maybe so?

```suggestion
    sealed class IntervalHandler :
        NpgsqlSimpleTypeHandler<Period>,
        INpgsqlSimpleTypeHandler<Duration>,
        INpgsqlSimpleTypeHandler<NpgsqlTimeSpan>,
        INpgsqlSimpleTypeHandler<TimeSpan>
```

Otherwise, remove the wrap.